engine/flink/management/periodic/src/main/scala/pl/touk/nussknacker/engine/management/periodic/PeriodicProcessService.scala

```diff
@@ -167,6 +173,16 @@ class PeriodicProcessService(
   }
 
   def handleFinished: Future[Unit] = {
+    def updateProcessAction(runningProcesses: Set[ProcessName])(
+        processName: ProcessName,
+        processActionIdOpt: Option[ProcessActionId]
+    ): Future[Boolean] = {
+      processActionIdOpt
+        .filter(_ => !runningProcesses(processName))
```

Comment:
Please rename this `runningProcesses` to sth like `stillActiveProcesses` or `processesWithRemainingRuns` or `stillScheduledProcesses`. Running here could be everyhing: running before `handleFisinhed` execution, some job deployed (RUNNING status) during computation of this method. Also, I'd rather like to keep this check near   `findActiveSchedulesForProcessesHavingDeploymentWithMatchingStatus` to be visible what's going on. Also, when we replace `DeploymentId` with `ProcessActionId`, the noise will be reduced.

Reply:
rescheduledProcesses?